pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -414,6 +496,19 @@ func (a *cpuAccumulator) sortAvailableCPUs() []int {
 	return result
 }
 
+// Sort all available CPUs:
+// - First by core using sortAvailableSockets().
+// - Then within each socket, sort cpus directly using the sort() algorithm defined above.
+func (a *cpuAccumulator) sortAvailableCPUsSpread() []int {
+	var result []int
+	for _, socket := range a.sortAvailableSockets() {
+		cpus := a.details.CPUsInSockets(socket).UnsortedList()
```

Comment:
why not just use `List()` instead of `UnsortedList()` ?

Reply:
Technically,  `List()` will give the same result here. `a.details` is built from topology information which gives kind of sorted CPU list.   `List()` introduce additional sorting cost, so I use `UnsortedList()` directly here.